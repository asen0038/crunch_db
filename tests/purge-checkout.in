set a 2 3 5
set b 8 4 7
set c 8 3 4
snapshot
del c
purge b
checkout 1
list snapshots
list entries
bye
