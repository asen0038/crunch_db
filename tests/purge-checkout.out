>
ok

>
ok

>
ok

>
saved as snapshot 1

>
ok

>
ok

>
ok

>
1

>
c [8 3 4]
a [2 3 5]

>
bye
